.circleci/config.yml

```diff
@@ -32,100 +24,117 @@ jobs:
           key: dep-cache-{{ checksum "Gopkg.lock" }}
           paths:
             - vendor
-      - run:
-          name: Install C dependencies
-          command: apt-get update && apt-get install -y build-essential libssl-dev uuid-dev squashfs-tools
+      - persist_to_workspace:
+          root: /go
+          paths:
+            - src/github.com/sylabs/singularity
+
+  go1.10-stretch:
+    <<: *defaults
+    docker:
+      - image: sylabsio/golang:1.10-stretch
+    steps:
+      - attach_workspace:
+          at: /go
       - run:
           name: Build Singularity
           command: |-
             ./mconfig -p /usr/local
             make -j$(nproc) -C ./builddir all
+
   go1.11-stretch:
     <<: *defaults
     docker:
-      - image: golang:1.11-stretch
+      - image: sylabsio/golang:1.11-stretch
     steps:
-      - checkout
-      - restore_cache:
-          key: dep-cache-{{ checksum "Gopkg.lock" }}
-          paths:
-            - vendor
-      - run:
-          name: Install dep
-          command: |-
-            go get -u github.com/golang/dep/cmd/dep
-      - run:
-          name: Install golint
-          command: |-
-            go get -u golang.org/x/lint/golint
-      - run:
-          name: Set Up Vendor Directory
-          command: |-
-            if [ ! -d vendor ]; then
-              dep ensure -vendor-only
-            fi
-      - save_cache:
-          key: dep-cache-{{ checksum "Gopkg.lock" }}
-          paths:
-            - vendor
-      - run:
-          name: Install C dependencies
-          command: apt-get update && apt-get install -y build-essential libssl-dev uuid-dev squashfs-tools
+      - attach_workspace:
+          at: /go
       - run:
           name: Build Singularity
           command: |-
             ./mconfig -p /usr/local
             make -j$(nproc) -C ./builddir all
-      - run: #Only run make check on go1.11 as it introduces a new formatting rule we're following
-          name: Run checks
-          command: make -C ./builddir check
+      - run:
+          name: check code
+          command: |-
+            go get -u golang.org/x/lint/golint
+            make -j$(nproc) -C ./builddir check
+
   go1.11-alpine:
     <<: *defaults
     docker:
-      - image: golang:1.11-alpine
+      - image: sylabsio/golang:1.11-alpine
     steps:
+      - attach_workspace:
+          at: /go
       - run:
-          name: Install git
+          name: Build Singularity
           command: |-
-            apk add --update git
+            ./mconfig -p /usr/local
+            make -j$(nproc) -C ./builddir all
+            go get -u golang.org/x/lint/golint
+            make -j$(nproc) -C ./builddir check
       - run:
-          name: checkout
+          name: check code
```

Comment:
personal linter says capitalize or dont capitalize :) but be consistent :)

Reply:
will fix